x-pack/test/functional/apps/reporting_management/report_delete_pagination.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'reporting']);
+  const log = getService('log');
+  const retry = getService('retry');
+  const security = getService('security');
+
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+
+  describe('Delete reports', function() {
+    before(async () => {
+      await security.testUser.setRoles(['global_discover_read', 'reporting_user']);
+      await esArchiver.load('empty_kibana');
+      await esArchiver.load('reporting/archived_reports');
+      await pageObjects.common.navigateToApp('reporting');
```

Comment:
I'm not sure how this works, since there is no "Reporting App". 

In the initial commits, the code was more clear that we're navigating to the Reporting section of the Management app, and I think it was better that way :) 

The idea of "Reporting app" should not be enforced in the code, because it implies Reporting ownership of the apps that use Reporting.

Reply:
reverted back  to navigateToActualURL() ;